### PR TITLE
Row stats

### PR DESCRIPTION
**Pull request by lilyclements**

### CODE REVIEW

**Comment by lilyclements** [comment]
General design edits @dannyparsons this is ready to merge

File: instat/dlgRowSummary.vb [hunk 1]
```diff
@@ -89,10 +59,38 @@ Public Class dlgRowSummary
         ucrSaveResults.SetPrefix("row_summary")
         ucrSaveResults.SetSaveTypeAsColumn()
         ucrSaveResults.SetDataFrameSelector(ucrSelectorForRowSummaries.ucrAvailableDataFrames)
-        ucrSaveResults.SetLabelText("Row Summary:")
+        ucrSaveResults.SetLabelText("Save Result:")
```
Comment (dannyparsons) [inline instat/dlgRowSummary.vb]: I think our default label here is "New Column Name"

Comment (lilyclements) [inline instat/dlgRowSummary.vb]: Fixed and synced in changes. This is ready to continue review

**Comment by maxwellfundi** [comment]
@dannyparsons this can be merged. No major changes here.

**Pull request merged**